wire-ios-request-strategy/Sources/Request Strategies/MLS/Actions/FetchMLSGroupInfoActionHandler.swift

```diff
@@ -40,15 +55,34 @@ class FetchPublicGroupStateActionHandler: ActionHandler<FetchPublicGroupStateAct
             return nil
```

Comment:
Shouldn't we also fail the action here with `.emptyParameters`?

Reply:
it's already the case